### ISSUE TITLE
Replace outdated markdown editor

### ISSUE THREAD

**Issue by christianlupus** [nextcloud/cookbook#991]
The currently used markdown core `v-markdown-editor` seems to no longer be maintained. We should switch to another markdown solution.

At best we would use a compatible code with other NC apps to have the same flavor.

**Comment by seyfeb** [comment]
[Nextcloud Notes](https://github.com/nextcloud/notes/blob/master/src/components/EditorMarkdownIt.vue) [uses](https://github.com/nextcloud/notes/blob/master/package.json) [markdown-it](https://www.npmjs.com/package/markdown-it) as does the [Markdown Editor](https://github.com/icewind1991/files_markdown/blob/master/package.json) and the [Tasks](https://github.com/nextcloud/tasks/blob/master/package.json) [app](https://github.com/nextcloud/tasks/blob/master/src/components/AppSidebar/NotesItem.vue). So I would suggest switching to the same.

**Comment by christianlupus** [comment]
The markdown-it is a parser of MArkdown as far as I know. So it takes a markdown text and renders it in HTML, right?

I was more about the editor part. We could use a plain text editor, can't we? We need to tweak it anyways to do the #903 topic.

**Comment by christianlupus** [comment]
Hello @harris2310. Thank you for your willingness to support the app.

It would be appreciated if you could provide a solution for the issue. Just comment that you are working on it and we will assign it to you (to prevent doubled work). I will be away in the next few days but you can start on it as you see fit. If you have questions or doubts, feel free to ask.

**Comment by harris2310** [comment]
Ima start working on it right now!

**Comment by christianlupus** [comment]
Hello @harris2310 did you succeed in getting this fix started? Is there anything missing that we can support you?

**Comment by harris2310** [comment]
Sorry christian, I won't be able to work on it cause I'll probably catch a job
My apologies.

**Comment by christianlupus** [comment]
No worries, we are not too active ATM.

**Comment by christianlupus** [comment]
I wrote a fix for the main problem. This is not zet the compatible version as suggested e.g. in [this comment](https://github.com/nextcloud/cookbook/issues/991#issuecomment-1126812601). The fix was just introduced to fix a security issue in NPM.

**Comment by christianlupus** [comment]
@MarcelRobitaille is this issue closed when the current PRs are merged? I just wanted to ask if there is still something open to be done.

**Comment by MarcelRobitaille** [comment]
> @MarcelRobitaille is this issue closed when the current PRs are merged? I just wanted to ask if there is still something open to be done.

I don't think so. I did not change the markdown editor, I only changed some things around it.

**Comment by christianlupus** [comment]
> I don't think so. I did not change the markdown editor, I only changed some things around it.

The editor I changed. This was the issue with the paste function we were discussing in the other issue/PR. I checked it once more. I will update the title as the topic is solved partially.

**Comment by MarcelRobitaille** [comment]
I don't think this needs to remain open for any of my PRs then.
